### PR TITLE
Add blpapi recipe

### PR DESCRIPTION
**Pull request by matthewgilbert**
Was hoping to add a recipe for `blpapi`. I'm fairly new to `conda` build recipes in general so any feedback is much appreciated. The python library depends on a C++ library so I was a bit unclear on the best way to package this for both windows and Linux.

### CODE REVIEW

File: recipes/blpapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec
+    folder: blpapi_cpp_windows
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python <=3.6
```
Comment (marcelotrevisani) [inline recipes/blpapi/meta.yaml]: Please remove this ``<=3.6``, in staged recipes we are using just python 2.7, 3.4, 3.5 and 3.6

File: recipes/blpapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+source:
+  - url: https://bloomberg.bintray.com/pip/simple/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 4c3cbf3fb63fd338360d55188f6a59568023be6e16531d4bd307ac59a297fd96
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-linux.tar.gz
+    sha256: c6c425f714b461ec618e6900bda8d9b4fc919892538ccfbf8c9a774e9e234fee
+    folder: blpapi_cpp_linux
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-windows.zip
+    sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec
+    folder: blpapi_cpp_windows
```
Comment (marcelotrevisani) [inline recipes/blpapi/meta.yaml]: if you really need to download files according to the platform, please use selectors like:
```
url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-linux.tar.gz  # [linux]
sha256: c6c425f714b461ec618e6900bda8d9b4fc919892538ccfbf8c9a774e9e234fee  # [linux]
url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-windows.zip  # [win]
sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec  # [win]
```

File: recipes/blpapi/meta.yaml (hunk 1 above)
Comment (matthewgilbert) [reply recipes/blpapi/meta.yaml]: Just to confirm, I should not include this even if I know `python=3.7` will break this?

File: recipes/blpapi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+
+test:
+  imports:
+    - blpapi
+
+about:
+  home: https://www.bloomberg.com/professional/support/api-library/
+  license: Copyright 2012. Bloomberg Finance L.P.
+  license_file: License.txt
```
Comment (marcelotrevisani) [inline recipes/blpapi/meta.yaml]: Could you please open a PR to this project to also add the ``License.txt`` in the package?

File: recipes/blpapi/build.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export BLPAPI_ROOT="."
+$PYTHON setup.py install
+
+if [ -d "$BLPAPI_ROOT/Linux" ]; then
+   cp -v $BLPAPI_ROOT/Linux/lib*.so $PREFIX/lib/
+else
+   cp -v $BLPAPI_ROOT/Darwin/lib*.so $PREFIX/lib/
```
Comment (marcelotrevisani) [inline recipes/blpapi/build.sh]: it would be good if the script installation of the project already do that

Comment (matthewgilbert) [reply recipes/blpapi/build.sh]: For this comment and your comment about related to the `License.txt` file when you are referring to the project you mean `blpapi`?

**Comment by matthewgilbert** [comment]
@marcelotrevisani Unsure if this is related to my build recipe or something more general with `conda-forge`, but the `appveyor` build seemed to pass but then was unsuccesful uploading the artifacts? At the end of [this](https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/19487136) it mentions
```
No artifacts found matching 'conda_packages\*.bz2' path
```
And in addition I cannot find a [windows build](https://anaconda.org/conda-forge/blpapi/files)

**Pull request merged**